crates/common/consensus/src/electra/beacon_state.rs

```diff
@@ -92,11 +93,37 @@ use crate::{
     withdrawal_request::WithdrawalRequest,
 };
 
+fn serialize_u8_as_quoted_list<S>(
+    value: &VariableList<u8, U1099511627776>,
+    serializer: S,
+) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    let string_vec: Vec<String> = value.iter().map(|v| v.to_string()).collect();
+    string_vec.serialize(serializer)
+}
+fn quoted_u8_var_list<'de, D>(deserializer: D) -> Result<VariableList<u8, U1099511627776>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let string_vec: Vec<String> = Vec::deserialize(deserializer)?;
+    let bytes = string_vec
+        .into_iter()
+        .map(|s| s.parse::<u8>().map_err(serde::de::Error::custom))
+        .collect::<Result<Vec<_>, _>>()?;
+    VariableList::new(bytes).map_err(|err| {
+        serde::de::Error::custom(format!("Cannot create VariableList from bytes {err:?}"))
+    })
+}
```

Comment:
```suggestion
pub mod quoted_u8_var_list {
    use super::*;

    pub fn serialize<S>(
        value: &VariableList<u8, U1099511627776>,
        serializer: S,
    ) -> Result<S::Ok, S::Error>
    where
        S: Serializer,
    {
        let string_vec: Vec<String> = value.iter().map(|v| v.to_string()).collect();
        string_vec.serialize(serializer)
    }

    pub fn deserialize<'de, D>(
        deserializer: D,
    ) -> Result<VariableList<u8, U1099511627776>, D::Error>
    where
        D: Deserializer<'de>,
    {
        let string_vec: Vec<String> = Vec::deserialize(deserializer)?;
        let bytes = string_vec
            .into_iter()
            .map(|s| s.parse::<u8>().map_err(D::Error::custom))
            .collect::<Result<Vec<_>, _>>()?;
        VariableList::new(bytes).map_err(|err| {
            D::Error::custom(format!("Cannot create VariableList from bytes: {err:?}"))
        })
    }
}

```

Replace the above with this so we can just do `#[serde(with = "quoted_u8_var_list")]`